docs/blog/_posts/2017-05-31-first-dotty-milestone-release.md

```diff
@@ -55,8 +55,8 @@ brew install lampepfl/brew/dotty
 ### Scastie
 
 [Scastie], the online Scala playground,
-supports Dotty.
-
+supports Dotty. 
+You can try it out there without installing.
```

Comment:
I would say "without installing anything", otherwise it looks like a word is missing from the sentence.